src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/scalable/ScoreVariantAnnotations.java

```diff
@@ -0,0 +1,190 @@
+import org.broadinstitute.hdf5.HDF5File;
+import org.broadinstitute.hdf5.HDF5LibException;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+import org.broadinstitute.hellbender.utils.python.PythonScriptExecutor;
+import org.broadinstitute.hellbender.utils.runtime.ProcessOutput;
+import picard.cmdline.programgroups.VariantFilteringProgramGroup;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * TODO
+ */
+@CommandLineProgramProperties(
+        // TODO
+        summary = "",
+        oneLineSummary = "",
+        programGroup = VariantFilteringProgramGroup.class
+)
+@DocumentedFeature
+public class ScoreVariantAnnotations extends VariantAnnotationWalker {
+
+    private static final String SCORER_PKL_SUFFIX = ".scorer.pkl";
+    private static final String SCORER_SER_SUFFIX = ".scorer.ser";
+    private static final String SCORES_HDF5_SUFFIX = ".scores.hdf5";
+    private static final String RECALIBRATION_VCF_SUFFIX = ".recal.vcf";
+
+    @Argument(
+            fullName = "python-script",
+            optional = true)
+    private File pythonScriptFile;
+
+    @Argument(
+            fullName = "model-prefix")
+    private String modelPrefix;
+
+    /**
+     * Add truth sensitivity slices through the call set at the given values. The default values are 100.0, 99.9, 99.0, and 90.0
+     * which will result in 4 estimated tranches in the final call set: the full set of calls (100% sensitivity at the accessible
+     * sites in the truth set), a 99.9% truth sensitivity tranche, along with progressively smaller tranches at 99% and 90%.
+     * Note: You must pass in each tranche as a separate value (e.g. -tranche 100.0 -tranche 99.9).
+     */
+    @Argument(
+            fullName = "truth-sensitivity-tranche",
+            shortName = "tranche",
+            doc = "The levels of truth sensitivity at which to slice the data. (in percent, that is 1.0 for 1 percent)",
+            optional = true)
+    private List<Double> truthSensitivityTranches = new ArrayList<>(Arrays.asList(100.0, 99.9, 99.0, 90.0));
+
+    private File inputScorerPklFile;
+    private File outputScoresHDF5File;
+    private PrintStream tranchesStream;
+
+    @Override
+    public String getVCFSuffix() {
+        return RECALIBRATION_VCF_SUFFIX;
+    }
+
+    @Override
+    public void beforeOnTraversalStart() {
+        isExtractAll = true;
+
+        if (pythonScriptFile != null) {
+            logger.info("Python script was provided, running in PYTHON mode...");
+
+            inputScorerPklFile = new File(modelPrefix + SCORER_PKL_SUFFIX);
+
+            IOUtils.canReadFile(pythonScriptFile);
+            IOUtils.canReadFile(inputScorerPklFile);
```

Comment:
You'll also want an error message if the Python script is provided but not the pickle and vice versa.